src/pages/technology/naming-system.md

```diff
@@ -0,0 +1,405 @@
+---
+title: Overview
+description: 'Blockstack naming system (BNS)'
```

Comment:
@GinaAbrams mentioned `BNS` should now be `Blockchain naming system`

Reply:
My understanding was that this name was kind of in flux, and that we should default to leaving the name as is until there's more finality.

Reply:
Updated in ca98406c